app/decorators/edit_request_decorator.rb

```diff
@@ -0,0 +1,37 @@
+class EditRequestDecorator < ApplicationDecorator
+  def status_label
+    if opened?
+      h.content_tag :span, "オープン", class: "label label-success"
+    elsif published?
+      h.content_tag :span, "公開済み", class: "label label-info"
+    elsif closed?
+      h.content_tag :span, "クローズ", class: "label label-danger"
+    end
+  end
+
+  def resource_diff_table
+    data = {
+      draft_resource: draft_resource,
+      diffs: draft_resource.diffs,
+      draft_values: draft_resource.draft_values,
+      origin_values: draft_resource.origin_values
+    }
+
+    h.render("db/application/resource_diff_table", data)
+  end
+
+  def edit_draft_resource_path
```

Comment:
Assignment Branch Condition size for edit_draft_resource_path is too high. [24.52/15]